server_tests/package.json

```diff
@@ -15,10 +15,11 @@
     "promise": "^7.0.3",
     "q": "^1.4.1",
     "request": "^2.55.0",
-    "rsvp": "^3.0.18"
+    "rsvp": "^3.0.18",
+    "standard": "^8.6.0"
   },
   "scripts": {
-    "test": "./node_modules/.bin/mocha --reporter spec",
+    "test": "standard && ./node_modules/.bin/mocha --reporter spec",
```

Comment:
I would rather make a "lint" script for this. I don't think we necessarily want to run this every time